bin/worker.js

```diff
@@ -240,7 +240,7 @@ async function main () {
   }
 
   // Build the listener and connect to the queue.
-  var taskListener = new TaskListener(runtime);
+  var taskListener = new TaskListener(host, runtime);
```

Comment:
I would suggest adding the host manager to the runtime in case later on this needs to be used elsewhere.